.github/workflows/conda.yml

```
@@ -38,6 +54,15 @@ jobs:
           use-mamba: true
           python-version: 3.9
           channel-priority: strict
+      - name: Remove outdated conda nightlies
+        if: github.event_name == 'push' && github.repository == 'dask/dask' && startsWith(github.ref, 'refs/tags/')
```

Do we want to check `is_tagged_commit` here as well? Or is that not needed for some reason?

---

> Or is that not needed for some reason?

At this point we should've already successfully prevented the race condition described in https://github.com/dask/dask/pull/11294#discussion_r1742366578, so think at this point the value of `is_tagged_commit` is interchangeable with that of `startsWith(github.ref, 'refs/tags/')`, such that we could represent this wirh something like

 ```suggestion
         if: github.event_name == 'push' && github.repository == 'dask/dask' && needs.check_tag.outputs.is_tagged_commit == 'true'
```

We should be good with what we have here, not sure if the suggestion would improve readability at all